cfn.properties

```diff
@@ -0,0 +1,21 @@
+name=Cloudformation Plugin for SonarQube
+category=External Analysers
+descripton=Cloudformation rules, supporting cfn-nag
+homepageUrl=https://github.com/Hack23/sonar-cloudformation-plugin
+archivedVersions=1.0.1
+publicVersions=1.0.2
+
+defaults.mavenGroupId=com.hack23.sonar
+defaults.mavenArtifactId=sonar-cloudformation-plugin
+
+1.0.2.description=Update dependency com.fasterxml.jackson.core:jackson-databind to v2.10.0.pr2
+1.0.2.date=2019-09-08
+1.0.2.sqVersions=[7.7,LATEST]
+1.0.2.changelogUrl=https://github.com/Hack23/sonar-cloudformation-plugin/releases/tag/sonar-cloudformation-plugin-1.0.2
+1.0.2.downloadUrl=https://oss.sonatype.org/service/local/repositories/releases/content/com/hack23/sonar/sonar-cloudformation-plugin/1.0.2/sonar-cloudformation-plugin-1.0.2.jar
+
+1.0.1.description=New rules added from cfn-nag
```

Comment:
There's not a lot of point in backfilling this old version, but you can if you want

Reply:
removed the old version, agree with you.